.github/workflows/release-pds-tools.yml

```
@@ -99,6 +99,7 @@ jobs:
           git restore sechub-api-java/src/main/resources/reduced-openapi3.json
           mkdir -p build/reports
           git status > build/reports/git-status.txt
+          git describe --tags >> build/reports/git-status.txt
```

This statement is imho not correct. Looks like it shows the latest tag.
For readability it would be great to print a separator line.
I suggest this:
```suggestion
          echo "--- git tags:" >> build/reports/git-status.txt
          git tag --points-at HEAD >> build/reports/git-status.txt
```